packages/admin/src/Resources/Resource.php

```diff
@@ -472,29 +477,29 @@ protected static function applyGlobalSearchAttributeConstraint(Builder $query, a
 
             $query->when(
                 method_exists($model, 'isTranslatableAttribute') && $model->isTranslatableAttribute($searchAttribute),
-                function (Builder $query) use ($databaseConnection, $searchAttribute, $searchOperator, $searchQuery, $whereClause): Builder {
+                function (Builder $query) use ($databaseConnection, $searchAttribute, $searchOperator, $searchQuery, $whereClause, $collation): Builder {
                     $searchColumn = match ($databaseConnection->getDriverName()) {
                         'pgsql' => "{$searchAttribute}::text",
-                        default => "json_extract({$searchAttribute}, '$')",
+                        default => "json_extract({$searchAttribute}, '$') {$collation}",
                     };
 
                     return $query->{"{$whereClause}Raw"}(
-                        "lower({$searchColumn}) {$searchOperator} ?",
+                        "lower({$searchColumn}) {$searchOperator} lower(?)",
```

Comment:
We cannot use lower(?) here since it has performance implications, is this really necessary?

Reply:
No, it can safely be omitted.